java/src/main/java/com/google/oak/transparency/LogEntryVerifier.java

```diff
@@ -40,6 +41,12 @@ public static void main(String[] args) throws Exception {
     byte[] publicKeyBytes = Files.readAllBytes(Path.of(args[1]));
     byte[] endorsementBytes = Files.readAllBytes(Path.of(args[2]));
 
+    // Auto-detect PEM format of public key.
+    String publicKeyString = new String(publicKeyBytes, StandardCharsets.UTF_8);
+    if (publicKeyString.startsWith("-----BEGIN PUBLIC KEY-----")) {
```

Comment:
Is this constant not defined anywhere that we already depend on?

Reply:
Couldn't find any lib for this. But at least avoiding repetition in many places.